recipe/meta.yaml

```diff
@@ -39,33 +39,36 @@ requirements:
     # llvm is needed for the headers
     - llvm-openmp              # [osx]
     - m2-patch  # [py==310 and win]
+    
 
   host:
     - python
     - pip
     - setuptools
     - wheel
-    - llvmlite 0.39.*
+    - llvmlite =0.40.0
     - numpy {{ numpy_min_ver }}
-    - tbb-devel 2021.*,<2021.6
+    - tbb-devel >=2021.6
     - _openmp_mutex            # [linux]
     - importlib_metadata       # [py<39]
 
   run:
     - python
     - importlib_metadata       # [py<39]
-    - llvmlite >=0.39.*,<0.40
+    - llvmlite >=0.40
     # NumPy has a hard upper limit.
     # NumPy 1.22.0, 1.22.1, 1.22.2 are all broken for ufuncs
     # xref: https://github.com/numba/numba/issues/7756
     # NumPy >=1.23 breaks with CUDA test: test_reinterpret_array_type
     # xref: https://github.com/numba/numba/issues/8529
-    - numpy >={{ numpy_min_ver }},!=1.22.0,!=1.22.1,!=1.22.2,<1.24
+    #- numpy >=1.21,!=1.22.0,!=1.22.1,!=1.22.2,<1.25
+    - numpy >=1.21 ,<1.25
     # needed for pkg_resources
     - setuptools
+    - libllvm14
```

Comment:
Should it be `libllvm14 >=14.0.6,<14.1.0a0` like here https://metayaml-conda.fly.dev/metayaml/reverse_dependencies?name=libllvm14 ?